### ISSUE TITLE
Graphic with image of text also has text too small for average user to read

### ISSUE THREAD

**Issue by ChrisXKeroack** [intel/openfl#856]
**Describe the bug**
Image of text in graphic is too small

**To Reproduce**
1. https://openfl.readthedocs.io/en/latest/running_the_federation.html 
2. Scroll to graphic at bottom of page

notice: Text too small (had to increase browser mag to 250%...), and insufficient color contrast on most text in center of graphic, and image of text instead of text in use, for director-based workflow graphic

**Expected behavior**
repeat a summary list of steps in text before/after the graphic

**Screenshots**
![image](https://github.com/securefederatedai/openfl/assets/104644211/74d369fb-6add-409d-9fb3-fe5ff3532368)

**Desktop:**
This issue is not OS or browser specific, but for tracking purposes, testing was done on Win11, in Edge Version 114.0.1823.58 (Official build) (64-bit).

**Smartphone:**
This issue is not device, OS, or browser specific.

**Additional context**
This is an accessibility issue, please annotate this bug with the 'accessibility' label.
